kafkazk/zookeeper.go

```diff
@@ -390,6 +341,37 @@ func (z *ZKHandler) GetTopics(ts []*regexp.Regexp) ([]string, error) {
 	return matchingTopics, nil
 }
 
+// GetTopicMetadata takes a topic name. If the topic exists, the topic metadata
+// is returned as a TopicMetadata.
+func (z *ZKHandler) GetTopicMetadata(t string) (TopicMetadata, error) {
+	topicMetadata := TopicMetadata{}
+
+	if t == "" {
+		return topicMetadata, fmt.Errorf("unspecified topic")
+	}
+
+	var path = "/brokers/topics/" + t
+	if z.Prefix != "" {
+		path = "/" + z.Prefix + path
+	}
```

Comment:
should that be factorized into a getPath() private struct method? it seems this is used a few times.

Reply:
Not a bad time to remove that, super old mess